content/docs/reactivesearch/searchbase/overview/apireference.md

```diff
@@ -56,6 +56,30 @@ const searchbase = new SearchBase(props);
     	weight: number;
     };
     ```
+-   **aggregationField** `string` [optional]
+    One of the most important use-cases this enables is showing `DISTINCT` results (useful when you are dealing with sessions, events and logs type data).
+    It utilizes `composite aggregations` which are newly introduced in ES v6 and offer vast performance benefits over a traditional terms aggregation.
+    You can read more about it over [here](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-composite-aggregation.html).
+    You can use `aggregationData` using `onAggregationData` callback or `subscriber` as shown:
+    ```javascript
+    const searchbase = new Searchbase({
+    	index: 'good-book-ds-latest',
+    	url: 'https://scalr.api.appbase.io',
+    	credentials: 'IPM14ICqp:8e573e86-8802-4a27-a7a1-4c7d0c62c186',
+    	dataField: 'original_title',
+    	aggregationField: 'original_title.keyword',
+    });
+    // using callback
+    searchbase.onAggregationData(next, prev) {}
+    // using subscriber
+    searchbase.subscribeToStateChanges(
```

Comment:
@ShahAnuj2610 You didn't define the `aggregations` key in the docs.
![](https://i.imgur.com/CCBxfBh.png)